homework.py

```diff
@@ -0,0 +1,41 @@
+import random
+from time import sleep
+from datetime import datetime
+def retry(Exceptions, tries,delay,backoff):
```

Comment:
Create a separate package with this decorator in it and then import `retry` here